src/functions.ts

```diff
@@ -73,52 +73,28 @@ const createFunctionFile = async (
   fnId: string,
   fnFilePath: string,
 ) => {
-  // Bundle File
-  let isImportMapPresent = false;
-  const importMapPath = `${options.workingDirectory}/import_map.json`;
-
-  try {
-    const { isFile } = await Deno.stat(importMapPath);
-    isImportMapPresent = isFile;
-  } catch (_e) {
-    isImportMapPresent = false;
-  }
+  const fnFileRelative = path.join("functions", `${fnId}.js`);
+  const fnBundledPath = path.join(options.outputDirectory, fnFileRelative);
 
-  let result;
   try {
-    result = await Deno.emit(fnFilePath, {
-      bundle: "module",
-      check: false,
-      importMapPath: isImportMapPresent ? importMapPath : undefined,
+    // call out to deno to handle bundling
+    const p = Deno.run({
+      cmd: [
+        "deno",
```

Comment:
looks like this would go out into $PATH and find deno again. Is that guaranteed to be the same deno that this current process is running? ex, what if my build hook is overridden to `path/to/special/deno ....`. 

overall seems pretty low risk, but if there's an easy workaround that'd be nice to fix. ex, in golang we use os.Executable() for a similar purpose. if not, a comment is probably OK.

Reply:
Ah thanks for noticing that - I added a call to `Deno.execPath()` which does that for us, and made sure it's not an unstable api 😄